www/docs/src/content/docs/start-here/getting-started.mdx

```diff
@@ -1,35 +1,65 @@
+title: Getting Started with StudioCMS
+description: Get started building with StudioCMS.
+sidebar:
+   order: 1
+   badge:
+     text: "Updated"
+     variant: "success"
 banner:
   content: |
     We've renamed the package to <b>studiocms</b> 🎉
 ---
 
 import { PackageManagers } from 'starlight-package-managers';
+import { Aside } from '@astrojs/starlight/components';
+import ReadMore from '~/components/ReadMore.astro';
+import StudioHeading from '~/components/StudioHeading.astro';
 
 ## Let's get started
 
-To start using Astro StudioCMS, you'll need:
+To start using StudioCMS, you'll need:
 
 - An Astro project
-- An Astro Studio account
-- An Astro Studio project
+- An Astro Studio account or (*Self*) Hosted libSQL server
 - The Astro StudioCMS integration
 
-## Creating an Astro Studio account
+## Understanding Options for `@astrojs/db`
 
-:::note
-Astro Studio is currently in closed beta. You'll need an invite to create an account.
+### Using libSQL
 
-We're following Astro Studio's News and Updates for the latest information on when the platform will be available to the public again and will be informing everyone through our [Discord channel](https://chat.astro-studiocms.xyz) when signups for Astro Studio are open again. There is also an open [PR for `@astrojs/db`](https://github.com/withastro/astro/pull/11385) that would allow you to use the Astro StudioCMS integration without the need for Astro Studio by using another LibSQL database service or even self-hosted.
-:::
+With the latest version of `@astrojs/db` (`v0.14.0`), you can now link to any libSQL server from any platform that exposes the libSQL remote protocol of the server, or can be self-hosted.
+
+To use libSQL with Astro Db, you will need to set the following environment variables:
+
+- **`ASTRO_DB_REMOTE_URL`** - The connection URL to your libSQL server
+- **`ASTRO_DB_APP_TOKEN`** - The app token for your libSQL server
+
+<ReadMore>For more information about these environment varibales, see [Using libSQL with @astrojs/db](/start-here/environment-variables/#if-using-libsql-with-astrojsdb)</ReadMore>
+
+The [commands for deploying and pushing changes](https://docs.astro.build/en/guides/astro-db/#deploy-with-a-studio-connection) to your database are the same when using libSQL as when connecting to an Astro Studio hosted database. However, both of your environment variables need to be set locally when running commands with the `--remote` option like `astro build` and `astro db push`.
+
+<ReadMore>See Astro's [Astro DB LibSQL Docs](https://docs.astro.build/en/guides/astro-db/#libsql) for more information</ReadMore>
+
+<StudioHeading>
+### Using Astro Studio
+</StudioHeading>
+
+<Aside type="note" title="Closed Beta">
+Astro Studio is currently in closed beta. You'll need an invite to create an account. For now if you do not have an account, you can use the [libSQL option](#using-libsql) to get started using another libSQL provider.
```

Comment:
We probably should say something more like "It's in closed beta, if you don't have an account you'll need to wait until signups are back" or whatever. Don't want people to think they can actually get an invite

Reply:
Updated the Latest version of the docs on [d869f86](https://github.com/astrolicious/studiocms/pull/281/commits/d869f869670c49fbd9c8f357a32ed15fe00a20c5)